zebra/zebra_nb_config.c

```diff
@@ -1216,32 +1218,50 @@ int lib_interface_zebra_bandwidth_destroy(struct nb_cb_destroy_args *args)
 }
 
 /*
- * XPath: /frr-vrf:lib/vrf/frr-zebra:ribs/rib
+ * XPath: /frr-vrf:lib/vrf/frr-zebra:zebra/ribs/rib
  */
-int lib_vrf_ribs_rib_create(struct nb_cb_create_args *args)
+int lib_vrf_zebra_ribs_rib_create(struct nb_cb_create_args *args)
 {
+	const char *vrf_name;
+	struct vrf *vrf;
+	afi_t afi = AFI_IP;
+	safi_t safi = SAFI_UNICAST;
+	struct zebra_vrf *zvrf;
+	struct zebra_router_table *zrt;
+
+	vrf_name = nb_running_get_entry(args->dnode, NULL, false);
+	vrf = vrf_lookup_by_name(vrf_name);
+	zvrf = vrf_info_lookup(vrf->vrf_id);
+
+	zrt = zebra_router_find_zrt(zvrf, zvrf->table_id, afi, safi);
```

Comment:
The afi and safi should be taken from the list entry that is being created (I believe the same holds true for the table-id).

Reply:
Ideally we do not accept to create RIB table creation via northbound. All tables are created during zebra init.
Waiting for identityref nortbound wrapper changes (which is coming in near term). Once the wrapper api to parse identityref is available will use it here.